### ISSUE TITLE
Right-clicking to pick tiles does not work

### ISSUE THREAD

**Issue by EmilEnglund** [defold/editor2-issues#1607]
I expect right-clicking a tile in the tile editor to pick that tile. Holding RMB should allow me marquee select a bunch of tiles to create a brush. Since editor1 had this feature I'm assuming that the lack of it in editor2 is a bug.

As a sidenote, an awesome addition to this would be to be able to marquee select and create brushes from the palette as well (the one that appears when I press SPACE).

**Comment by subsoap** [comment]
Check out using Tiled for now. You can export .tilemap files with it.

**Comment by EmilEnglund** [comment]
Yeah, Tiled is great! That's where these feature ideas came from in the first place :)

Right now however, I'm more inclined to use the old editor rather than add an extra tool into my workflow.

**Comment by rgrams** [comment]
The feature is not gone, hold Shift and left click/drag to select tiles.
